src/components/link/common/link.scss

```diff
@@ -5,57 +5,25 @@
 @include sbb.host-component-properties;
 
 :host {
-  // Necessary to forward styles to native link
-  @include sbb.text-inherit;
-
-  display: inline-block;
-  color: inherit;
-
   // Use !important here to not interfere with Firefox focus ring definition
   // which appears in normalize css of several frameworks.
   outline: none !important;
-
-  @include sbb.link-variables;
-
-  --sbb-link-gap: var(--sbb-spacing-fixed-1x);
-}
-
-:host([variant='block']) {
-  display: block;
-}
-
-// Using :where to be not more specific than inline variant
-:host([negative]) {
-  @include sbb.link-variables--negative;
-}
-
-:host([variant='inline']) {
-  @include sbb.link-variables--inline;
-}
-
-:host([variant='inline'][negative]) {
-  @include sbb.link-variables--inline-negative;
 }
 
 :host([role='button']) {
```

Comment:
I believe it's not worth it :/
I would need to create the CSS files for the button variants and more than that, I would need to import 3 scss files inside the components (which I think might lead to an increase in bundle size since they are converted to a string constant when compile. Not 100% sure though)